server/interpreter.js

```diff
@@ -3058,7 +3054,7 @@ Interpreter.prototype['stepSequenceExpression'] = function(stack, state, node) {
   var expression = node['expressions'][n];
   if (expression) {
     state.n_ = n + 1;
-    this.pushNode_(expression);
+    return new Interpreter.State(expression, state.scope);
   } else {
```

Comment:
I've done a pass through all the step functions to rationalise this.  Quite a few `else`es have been removed.